lib/board.py

```diff
@@ -150,13 +150,53 @@ def all_visible_except_bombs(self) -> bool:
 
         return True
 
+
+    def get_top_guide_rows(self) -> List[str]:
+        """
+            returns top rows with column index
```

Comment:
[nit] Capitalize first letter

Reply:
Add note, e.g. in docstring, that this can only handle up to 2 digits (This limit is entirely reasonable, but would be nice to declare it nontheless)